src/store.js

```diff
@@ -0,0 +1,32 @@
+/**
+ * Copyright (c) Hathor Labs and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+export default class MemoryStore {
```

Comment:
I think we should move it to the `hathor-walelt-lib`.

Reply:
Same as this https://github.com/HathorNetwork/hathor-explorer/pull/4#discussion_r334080351